packaging/docker/entrypoint.py

```diff
@@ -2,17 +2,54 @@
 import json
 import logging
 import os
+import shutil
 import subprocess
 import time
+from datetime import datetime, timedelta
+from glob import glob
 from pathlib import Path
-from typing import Dict, Optional, Any, List
+from typing import Any, Dict, List, Optional
 
 logger = logging.getLogger('monitor')
 logging.basicConfig(level=logging.DEBUG)
 
 DEFAULT_LOG_LEVEL = 'critical'
 
 
+def can_delete(file: str, cutoff: int) -> bool:
+    return int(os.stat(file).st_mtime) <= cutoff or Path(file).name.startswith('_MEI')
+
+
+def cleanup_tmp() -> None:
+    logger.info('Preparing to cleanup tmp directory')
+    tmp_dir = glob('/tmp/*')
```

Comment:
Since you are already using the newer pathlib why not use its glob function? https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob